core/control-panel/impl/src/core/constants.rs

```diff
@@ -25,7 +25,7 @@ pub const CANISTER_CONFIG_STATE_SIZE: u32 = WASM_PAGE_SIZE * CANISTER_CONFIG_TOT
 pub const MAX_WASM_PAGES: u64 = MAX_STABLE_MEMORY_SIZE / WASM_PAGE_SIZE as u64;
 
 /// The initial cycle balance to set for new station canisters.
-pub const INITIAL_STATION_CYCLES: u128 = 2_500_000_000_000;
+pub const INITIAL_STATION_CYCLES: u128 = 5_000_000_000_000;
```

Comment:
Let's move this down to 4TC, it would make the distribution of initial remaining cycles per canister as:

- Fiduciary subnet:
  - Station:  0.7 TC, idle cycle consumption right after the bootstrap is 0.053TC month
  - Upgrader:  0.8 TC, idle cycle consumption right after the bootstrap is 0.038TC month

- Application subnet:
  - Station:  1.5 TC, idle cycle consumption right after the bootstrap is 0.02TC month
  - Upgrader:  1.5 TC, idle cycle consumption right after the bootstrap is 0.013TC month